content/sql/alter-column-type.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Change Column Type"
+description: Template to modify a column type. Make sure to change the name and type.
+author: Mihai
+github: https://github.com/mihaiandrei97
+---
+
+```sql
+alter table table_name
+    alter COLUMN column_name type new_data_type;

```

Comment:
Could you make the code consistent by lowercasing all the script? 😀

Reply:
Done! Sorry, first time the keywords were uppercase, and after that i checked the other files and changed them to lowercase, but i missed one :D

Reply:
Thank you for your contribution! It would be great if you go through those smaller SQL script, and add more info, and example to them! 😉